knexfile.js

```diff
@@ -1,41 +1,50 @@
-const dbConnection = process.env.DATABASE_URL;
+require("dotenv").config();
+
+const localPgConnection = {
+  host: process.env.DB_HOST,
+  database: process.env.DB_DATABASE,
+  user: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  port: process.env.DB_PORT
+};
+
+const dbConnection = process.env.DATABASE_URL || localPgConnection;
 
 module.exports = {
-  development: {
```

Comment:
Are we supposed to be commenting out the dev and testing environments? Forgive me for my ignorance if this is a silly question ⚖️

Reply:
Oh.  No.  Sorry.  I forgot to uncomment those.  I'll get them back in right now.

Reply:
Awesome, thank you. Would you mind giving the rest of your changed files a quick look? There's a lot of commented out code there and I'm not familiar enough with Node to determine if it's necessary or not